### ISSUE TITLE
Use postuninstall npm script to clean up any toolbelt files

### ISSUE THREAD

**Issue by firstdoit** [vtex/toolbelt#132]
https://docs.npmjs.com/misc/scripts

To avoid this:

```
guilherme@trinsic ~/P/V/toolbelt (feature/request-retry)> npm uninstall -g vtex
unbuild vtex@0.19.2
guilherme@trinsic ~/P/V/toolbelt (feature/request-retry)> ls ~/.vtex
credentials.json
```

**Comment by brenoc** [comment]
That's good!

**Comment by firstdoit** [comment]
Fixed via 3d74778
